port/esp32/main/main.c

```diff
@@ -486,7 +419,8 @@ obtain_time(void)
   }
 }
 
-#define STACK_SIZE (8 * 1024)
+// #define STACK_SIZE (8 * 1024)
```

Comment:
is it possible to have it on 8kB ? Or why it is increased ?

Reply:
I think that was just so I can enable debug logs, it should work with 8kB, I will test it.

Reply:
Maybe add `#ifdef` when debug logs are enabled.